DISPATCH_SUBSCRIBE.md

```diff
@@ -0,0 +1,98 @@
+# Forensics on RDX usage
+
+This is a "working" guess on how RDX works, and how to implement it into your own system.
```

Comment:
These are two different things (how it works, vs how to use it). I think it would make more sense to focus on the latter (i.e. how to use it to build apps)